### PR TITLE
Add recipe for Phylommand

### PR DESCRIPTION
**Pull request by brendanf**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/phylommand/build.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# CFLAGS, CPPFLAGS, and LDFLAGS are missing
+sed -i.bak '/\$(PP).*-c/ s/-c/$(CFLAGS) -c/' Makefile
+sed -i.bak '/\$(CC).*-c/ s/-c/$(CFLAGS) -c/' Makefile
+sed -i.bak '/\$(PP).*-o/ s/-o/$(LDFLAGS) -o/' Makefile
+
+export CFLAGS="-I$PREFIX/include"
+export LDFLAGS="-L$PREFIX/lib"
+
+if [[ "$(uname)" == "Linux" ]] ; then
+    PP="$GXX -std=c++11"
```
Comment (epruesse) [inline recipes/phylommand/build.sh]: Use `$CXXFLAGS` - that includes the correct std settings (conda builds with c++17)

Comment (epruesse) [reply recipes/phylommand/build.sh]: Also, doesn't need a switch. Just use `PP="$CXX $CXXFLAGS"`

File: recipes/phylommand/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  url: https://zenodo.org/record/200397/files/phylommand_v1.0.zip
+  md5: 14ae8509ff838530dcb50e7fb3ca48c1
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```
Comment (epruesse) [inline recipes/phylommand/meta.yaml]: Needs both?

File: recipes/phylommand/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  host:
+    - nlopt
+  run:
+    - nlopt
+
+about:
+  home: https://github.com/mr-y/phylommand
+  license: GPLv3
+  license_file: LICENSE
+  summary: Command-line tools to create, manipulate, and/or analyze phylogenetic trees or pairwise alignments. Does not include "rudisvg" svg viewer.
```
Comment (epruesse) [inline recipes/phylommand/meta.yaml]: Try to be more brief - templates look better with 1 line for `summary` and details in `description`.

File: recipes/phylommand/meta.yaml (hunk 2 above)
Comment (brendanf) [reply recipes/phylommand/meta.yaml]: Yes, there's one C file.

File: recipes/phylommand/build.sh (hunk 1 above)
Comment (brendanf) [reply recipes/phylommand/build.sh]: Great, thanks for the help!

**Comment by epruesse** [comment]
@biocodnabot recheck

**Comment by epruesse** [comment]
@biocondabot recheck

**Comment by epruesse** [comment]
@biocondabot merge

**Comment by epruesse** [comment]
Thanks @brendanf

**Pull request merged**